source/CAS/Client.php

```diff
@@ -2595,11 +2595,7 @@ private function _callback()
             if (preg_match('/^[PT]GT-[\.\-\w]+$/', $pgtId)) {
                 phpCAS::trace('Storing PGT `'.$pgtId.'\' (id=`'.$pgtIou.'\')');
                 $this->_storePGT($pgtId, $pgtIou);
-                if (array_key_exists('HTTP_ACCEPT', $_SERVER) &&
-                    (   $_SERVER['HTTP_ACCEPT'] == 'application/xml' ||
-                        $_SERVER['HTTP_ACCEPT'] == 'text/xml'
-                    )
-                ) {
+                if (array_key_exists('HTTP_ACCEPT', $_SERVER) && $this->isXmlResponse()) {
```

Comment:
I would move the `array_key_exists('HTTP_ACCEPT', $_SERVER)` check to `isXmlResponse()`.